sql/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -97,7 +91,7 @@
     private final List<Channel> serverChannels = new ArrayList<>();
     private final List<InetSocketTransportAddress> boundAddresses = new ArrayList<>();
     @Nullable
-    private  BoundTransportAddress boundAddress = null;
+    private BoundTransportAddress boundAddress = null;
```

Comment:
Since you changed this already, you might want to remove the `= null` which is not necessary for fields.